DarkFlippers/unleashed-firmware
# feat[unirfremix]: allow to send dynamic protocols

# What's new

- Currently you can only send RAW .sub files in the "SubGhz Remote" App. This PR should allow to send other protocols (e.g. Princeton or Keeloq).

# Verification 

- Create a Keeloq .sub file (`SubGhz` -> `Add Manually` -> `Doorhan_433`)
- Create a Princeton .sub file (`SubGhz` -> `Add Manually` -> `Princeton_433`)
- Record a RAW .sub file (`SubGhz` -> `Read RAW` -> `Save`)
- *You can also try and test other protocols not listed*
- Create a map file for UniRFRemix and add the .sub files to it
- Try to send all .sub files and then see if `Cnt` for Doorhan increases and verify that all .sub files were sent correctly

# Checklist (For Reviewer)
